src/distribution.cpp

```diff
@@ -381,14 +362,16 @@ UPtrDist distribution_from_xml(pugi::xml_node node)
     dist = UPtrDist {new Watt(node)};
   } else if (type == "normal") {
     dist = UPtrDist {new Normal(node)};
-  } else if (type == "muir") {
-    dist = UPtrDist {new Muir(node)};
   } else if (type == "discrete") {
     dist = UPtrDist {new Discrete(node)};
   } else if (type == "tabular") {
     dist = UPtrDist {new Tabular(node)};
   } else if (type == "mixture") {
     dist = UPtrDist {new Mixture(node)};
+  } else if (type == "muir") {
+    openmc::fatal_error(
+      "'muir' distribution type is no longer supported. Please regenerate your "
+      "XML files using the Python API.");
```

Comment:
```suggestion
    openmc::fatal_error(
      "'muir' distribution are now specified using normal distributions. Please "
      "regenerate your XML files using the Python API.");
```

Does this suggestion on suggestion on suggestion this cover everything :smile:

Reply:
I'm a fan @paulromano 👍🏻

Reply:
Ok, just updated it